packages/@aws-cdk/aws-redshift/lib/cluster.ts

```diff
@@ -142,6 +142,19 @@ export interface ICluster extends IResource, ec2.IConnectable, secretsmanager.IS
    * @attribute EndpointAddress,EndpointPort
    */
   readonly clusterEndpoint: Endpoint;
+
+  /**
+   * Adds a parameter to the Clusters' parameter group
+   *
+   * @param name the parameter name
+   * @param value the parameter name
+   * @returns metadata about the execution of this method. If the parameter
+   * was not added, the value of `parameterAddedResult` will report a failure status. You
+   * should always check this value to make sure that the operation was
+   * actually carried out. Otherwise, synthesis and deploy will terminate
+   * silently, which may be confusing.
+   */
+  addToParameterGroup(name: string, value: string): AddParameterResult;
```

Comment:
This is not idiomatic for CDK at all, and moreover this is the wrong user experience. This should throw an error if the parameter was not added because the only reason a user would call this method is if they wanted the parameter to be added; the parameter not being added indicates that they have misconfigured something, and their synthesis should fail so that they know.